tests/performance_tests/performance_test_basic.py

```diff
@@ -365,8 +366,16 @@ def runTpsTest(self) -> PtbTpsTestResult:
         completedRun = False
         self.producerNode = self.cluster.getNode(self.producerNodeId)
         self.connectionPairList = []
-        for producerId in self.clusterConfig._producerNodeIds:
-            self.connectionPairList.append(f"{self.cluster.getNode(producerId).host}:{self.cluster.getNodeP2pPort(producerId)}")
+
+        def configureConnections():
+            if(self.ptbConfig.endpointApi == "http"):
+                for apiNodeId in self.clusterConfig._apiNodeIds:
+                    self.connectionPairList.append(f"{self.cluster.getNode(apiNodeId).host}:{self.cluster.getNodeHttpPort(apiNodeId)}")
```

Comment:
Instead of adding `cluster.getNodeHttpPort`, why is this not `self.cluster.getNode(apiNodeId).port`, like `.host`

Reply:
Addressed: [f342f5e](https://github.com/AntelopeIO/leap/pull/1138/commits/f342f5e3cace6bd7db29e548640fc9a23fe53d43)